test/fixtures/output/service-quota/1.golden

```diff
@@ -1,6 +1,6 @@
   Quota Code |  Name   |     Scope     | Applied Limit | Usage | Organization | Environment | Kafka Cluster | Network | User  
 -------------+---------+---------------+---------------+-------+--------------+-------------+---------------+---------+-------
-  quota_a    | Quota A | kafka_cluster |            15 |    10 | org-123      | env-1       | lkc-1         |         |       
```

Comment:
`Applied Limit` is `int32` in the `quotaValue` struct, so shouldn't it also be left-aligned here? Same w/ `Partition`, `Partition Mirror Lag`, etc. in the mirror commands.